### PR TITLE
updating extensions and core packages to latest

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

File: src/WebJobs.Script/WebJobs.Script.csproj [hunk 1]
```diff
@@ -63,43 +63,43 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.WebJobs, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Core.2.0.0-alpha-10359\lib\net45\Microsoft.Azure.WebJobs.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Core.2.0.0-alpha-10360\lib\net45\Microsoft.Azure.WebJobs.dll</HintPath>
```
Comment (mathewc) [inline src/WebJobs.Script/WebJobs.Script.csproj]: did you update the nuspecs of the two packages in this sln?

**Comment by mathewc** [comment]
Just make sure nuspec refs are updated as well, otherwise looks good.

File: src/WebJobs.Script/WebJobs.Script.csproj (hunk 1 above)
Comment (brettsam) [inline src/WebJobs.Script/WebJobs.Script.csproj]: Yeah good point. I'll pull the nuspec checker in here later today.

**Comment by mathewc** [comment]
:ship:

**Pull request merged**